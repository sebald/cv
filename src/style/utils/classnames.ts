export const mergeClassNames = (...names: (string | undefined)[]) => ({
  className: names
    .filter(x => typeof x === 'string')
    .join(' ')
    .trim(),
});
