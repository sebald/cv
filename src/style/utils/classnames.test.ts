import { mergeClassNames } from './classnames';

test('merge classnames', () => {
  expect(mergeClassNames('foo', 'bar', undefined)).toMatchSnapshot();
});
