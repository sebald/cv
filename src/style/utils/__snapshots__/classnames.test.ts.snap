// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`merge classnames 1`] = `
Object {
  "className": "foo bar",
}
`;
